tests/Config/Loader/PhpFileLoaderTest.php

```diff
@@ -93,14 +98,83 @@ public function testLoad()
 EOF;
 
         $this->assertEquals(
-            $content,
+            $expects,
             $this->loader->load($this->getRootDir().'/vendor/contao/test-bundle/Resources/contao/dca/tl_test.php')
         );
+    }
+
+    /**
+     * Test loading a file with a custom namespace.
+     */
+    public function testLoadNamespace()
+    {
+        $expects = <<<'EOF'
+
+namespace Foo\Bar {
+//namespace Foo\Bar;
+
+$GLOBALS['TL_DCA']['tl_test']['config']['dataContainer'] = 'DC_Table';
+}
+
+EOF;
+
+        $this->assertEquals(
+            $expects,
+            $this->loader->load(
+                $this->getRootDir().'/vendor/contao/test-bundle/Resources/contao/dca/tl_namespace.php',
+                PhpFileLoader::NAMESPACED
+            )
+        );
+    }
+
+    /**
+     * Test loading a file with a declare statement.
+     */
+    public function testLoadDeclare()
+    {
+        $expects = <<<'EOF'
+
+/**
+ * I am a declare(strict_types=1) comment
+ */
+
+//declare(strict_types=1);
+
+$GLOBALS['TL_DCA']['tl_test']['config']['dataContainer'] = 'DC_Table';
+
+EOF;
+
+        $this->assertEquals(
+            $expects,
+            $this->loader->load($this->getRootDir().'/vendor/contao/test-bundle/Resources/contao/dca/tl_declare.php')
+        );
+
+        $expects = <<<'EOF'
+
+//declare(strict_types  =   1,ticks = 1);
```

Comment:
This is not the correct behaviour. It needs to be `declare(ticks=1);`. Only strip `strict_types=1`.